shared/whatYouNeed/raspberrypi3.md

```diff
@@ -1,8 +1,8 @@
 <img style="float: right;padding-left: 10px;" src="/img/raspberrypi/raspberrypi.jpg" width="25%">
 * A [Raspberry Pi 3 model B][rpi3B]. See our [supported devices list][supportedDevicesList] for other boards.
 * A 4GB or larger microSD card. The [speed class][sdSpeed] of the card also matters - class 10 card or above is the way to go.
-* **[Optional]** An ethernet cable
 * A micro USB cable.
+* **[Optional]** An ethernet cable.
 * **[Optional]** A [2A USB micro power supply][psu].
```

Comment:
Given the high-power-draw of the rpi3, perhaps we shouldn't label a 2A PSU as "optional" ?